packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -733,7 +733,7 @@ curl -v -X POST \
 | key      | The header name for the authorization header                                  | String     | TRUE       | FALSE    |
 | value    | The header value                                                              | String     | TRUE       | TRUE     |
```

Comment:
Let's use the same Description for the "value" property as we do in Event Hooks:
https://developer.okta.com/docs/reference/api/event-hooks/#authscheme-object

BEFORE: The header value
AFTER: The header value. This is the secret value that you want Okta to pass to your external service endpoint (so that your external service can check for its presence as a security measure).

Reply:
Good catches. Updated as per comments.